### ISSUE TITLE
[Bug] C++ SDK load TorchScript model error  Unknown builtin op: torchvision::roi_align 加载TorchScript模型报错

### ISSUE THREAD

**Issue by 1095788063** [open-mmlab/mmdeploy#1642]
### Checklist

- [X] I have searched related issues but cannot get the expected help.
- [X] 2. I have read the [FAQ documentation](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/faq.md) but cannot get the expected help.
- [X] 3. The bug has not been fixed in the latest version.

### Describe the bug

[2023-01-11 23:44:08.926] [mmdeploy] [error] [torch_net.cpp:161] unhandled exception:
Unknown builtin op: torchvision::roi_align.
Could not find any similar ops to torchvision::roi_align. This op may not exist or may not be currently supported in TorchScript.
:
D:\Python\Python38\lib\site-packages\torchvision\ops\roi_align.py(61): roi_align
D:\Python\Python38\lib\site-packages\mmcv\ops\roi_align.py(200): forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1098): _slow_forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1110): _call_impl
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\codebase\mmdet\models\roi_heads\single_level_roi_extractor.py(239): single_roi_extractor__forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1098): _slow_forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1110): _call_impl
d:\ayjdata\code\deep_learning\openmmlab\mmdetection\mmdetection-v3.0.0rc5_dev_2023-01-09\mmdet\models\roi_heads\standard_roi_head.py(158): _bbox_forward
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\codebase\mmdet\models\roi_heads\standard_roi_head.py(51): standard_roi_head__predict_bbox
d:\ayjdata\code\deep_learning\openmmlab\mmdetection\mmdetection-v3.0.0rc5_dev_2023-01-09\mmdet\models\roi_heads\base_roi_head.py(118): predict
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\codebase\mmdet\models\detectors\two_stage.py(92): two_stage_detector__forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1098): _slow_forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1110): _call_impl
D:\Python\Python38\lib\site-packages\torch\jit\_trace.py(958): trace_module
D:\Python\Python38\lib\site-packages\torch\jit\_trace.py(741): trace
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\torch_jit\trace.py(108): trace
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(107): __call__
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(275): call_function_local
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(326): call_function
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(356): _wrap
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\pytorch2torchscript.py(63): torch2torchscript
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(107): __call__
D:\Python\Python38\lib\multiprocessing\process.py(108): run
D:\Python\Python38\lib\multiprocessing\process.py(315): _bootstrap
D:\Python\Python38\lib\multiprocessing\spawn.py(129): _main
D:\Python\Python38\lib\multiprocessing\spawn.py(116): spawn_main
<string>(1): <module>
Serialized   File "code/__torch__/mmcv/ops/roi_align.py", line 9
    boxes: Tensor,
    argument_2: Tensor) -> Tensor:
    value = ops.torchvision.roi_align(argument_2, boxes, 0.25, 7, 7, 0, True)
            ~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    return value

![Uploading image.png…]()


### Reproduction

win10 C++ SDK 加载  mask TorchScript  模型报错

### Environment

```Shell
win10 C++ SDK 加载  mask TorchScript  模型报错
```


### Error traceback

```Shell
win10 C++ SDK 加载  mask TorchScript  模型报错
```

**Comment by 1095788063** [comment]
loading mmdeploy_execution ...
loading mmdeploy_cpu_device ...
loading mmdeploy_cuda_device ...
loading mmdeploy_graph ...
loading mmdeploy_directory_model ...
loading mmdeploy_zip_model ...
loading mmdeploy_operation ...
loading mmdeploy_operation_cpu ...
loading mmdeploy_operation_cuda ...
loading mmdeploy_operation_dummy ...
loading mmdeploy_transform ...
loading mmdeploy_elena_transform_impl ...
loading mmdeploy_transform_module ...
loading mmdeploy_trt_net ...
loading mmdeploy_torch_net ...
loading mmdeploy_net_module ...
loading mmdeploy_mmdet ...
[2023-01-11 23:44:08.343] [mmdeploy] [info] [model.cpp:35] [DirectoryModel] Load model: "D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdetection\mmdetection-V3.0.0rc5_dev_2023-01-09\runs\train\mask-rcnn_r50_fpn_2x_coco_4CH-2023-01-10\torchscript"
[2023-01-11 23:44:08.343] [mmdeploy] [info] [model.cpp:35] [DirectoryModel] Load model: "D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdetection\mmdetection-V3.0.0rc5_dev_2023-01-09\runs\train\mask-rcnn_r50_fpn_2x_coco_4CH-2023-01-10\torchscript"
[2023-01-11 23:44:08.513] [mmdeploy] [info] [inference.cpp:44] {
  "context": {
    "device": "<any>",
    "model": "<any>",
    "stream": "<any>"
  },
  "pipeline": {
    "input": [
      "img"
    ],
    "output": [
      "post_output"
    ],
    "tasks": [
      {
        "input": [
          "img"
        ],
        "module": "Transform",
        "name": "Preprocess",
        "output": [
          "prep_output"
        ],
        "transforms": [
          {
            "file_client_args": {
              "backend": "disk"
            },
            "type": "LoadImageFromFile"
          },
          {
            "keep_ratio": true,
            "size": [
              640,
              640
            ],
            "type": "Resize"
          },
          {
            "mean": [
              123.675,
              116.28,
              103.53
            ],
            "std": [
              58.395,
              57.12,
              57.375
            ],
            "to_rgb": true,
            "type": "Normalize"
          },
          {
            "size_divisor": 32,
            "type": "Pad"
          },
          {
            "type": "DefaultFormatBundle"
          },
          {
            "keys": [
              "img"
            ],
            "meta_keys": [
              "img_path",
              "pad_param",
              "img_shape",
              "valid_ratio",
              "scale_factor",
              "pad_shape",
              "flip",
              "ori_filename",
              "flip_direction",
              "img_id",
              "ori_shape",
              "img_norm_cfg",
              "filename"
            ],
            "type": "Collect"
          }
        ],
        "type": "Task"
      },
      {
        "input": [
          "prep_output"
        ],
        "input_map": {
          "img": "#0"
        },
        "is_batched": false,
        "module": "Net",
        "name": "maskrcnn",
        "output": [
          "infer_output"
        ],
        "output_map": {
          "dets": "#0",
          "labels": "#1",
          "masks": "#2"
        },
        "type": "Task"
      },
      {
        "component": "ResizeInstanceMask",
        "input": [
          "prep_output",
          "infer_output"
        ],
        "module": "mmdet",
        "name": "postprocess",
        "output": [
          "post_output"
        ],
        "params": {
          "mask_thr_binary": 0.5,
          "min_bbox_size": 0,
          "rcnn": {
            "mask_thr_binary": 0.5,
            "max_per_img": 100,
            "nms": {
              "iou_threshold": 0.5,
              "type": "nms"
            },
            "score_thr": 0.05
          },
          "rpn": {
            "max_per_img": 1000,
            "min_bbox_size": 0,
            "nms": {
              "iou_threshold": 0.7,
              "type": "nms"
            },
            "nms_pre": 1000
          },
          "score_thr": 0.05
        },
        "type": "Task"
      }
    ]
  }
}
[2023-01-11 23:44:08.926] [mmdeploy] [error] [torch_net.cpp:161] unhandled exception:
Unknown builtin op: torchvision::roi_align.
Could not find any similar ops to torchvision::roi_align. This op may not exist or may not be currently supported in TorchScript.
:
D:\Python\Python38\lib\site-packages\torchvision\ops\roi_align.py(61): roi_align
D:\Python\Python38\lib\site-packages\mmcv\ops\roi_align.py(200): forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1098): _slow_forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1110): _call_impl
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\codebase\mmdet\models\roi_heads\single_level_roi_extractor.py(239): single_roi_extractor__forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1098): _slow_forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1110): _call_impl
d:\ayjdata\code\deep_learning\openmmlab\mmdetection\mmdetection-v3.0.0rc5_dev_2023-01-09\mmdet\models\roi_heads\standard_roi_head.py(158): _bbox_forward
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\codebase\mmdet\models\roi_heads\standard_roi_head.py(51): standard_roi_head__predict_bbox
d:\ayjdata\code\deep_learning\openmmlab\mmdetection\mmdetection-v3.0.0rc5_dev_2023-01-09\mmdet\models\roi_heads\base_roi_head.py(118): predict
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\codebase\mmdet\models\detectors\two_stage.py(92): two_stage_detector__forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1098): _slow_forward
D:\Python\Python38\lib\site-packages\torch\nn\modules\module.py(1110): _call_impl
D:\Python\Python38\lib\site-packages\torch\jit\_trace.py(958): trace_module
D:\Python\Python38\lib\site-packages\torch\jit\_trace.py(741): trace
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\torch_jit\trace.py(108): trace
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(107): __call__
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(275): call_function_local
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(326): call_function
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(356): _wrap
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\pytorch2torchscript.py(63): torch2torchscript
D:\ayjdata\Code\Deep_learning\OpenMMLab\mmdeploy\mmdeploy-dev-V1.0.0rc1-V20230111\mmdeploy\apis\core\pipeline_manager.py(107): __call__
D:\Python\Python38\lib\multiprocessing\process.py(108): run
D:\Python\Python38\lib\multiprocessing\process.py(315): _bootstrap
D:\Python\Python38\lib\multiprocessing\spawn.py(129): _main
D:\Python\Python38\lib\multiprocessing\spawn.py(116): spawn_main
<string>(1): <module>
Serialized   File "code/__torch__/mmcv/ops/roi_align.py", line 9
    boxes: Tensor,
    argument_2: Tensor) -> Tensor:
    value = ops.torchvision.roi_align(argument_2, boxes, 0.25, 7, 7, 0, True)
            ~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
    return value

[2023-01-11 23:44:08.943] [mmdeploy] [error] [torch_net.cpp:226] Failed to created TorchNet with config: {
  "context": {
    "device": "<any>",
    "model": "<any>",
    "stream": "<any>"
  },
  "input": [
    "prep_output"
  ],
  "input_map": {
    "img": "#0"
  },
  "is_batched": false,
  "module": "Net",
  "name": "maskrcnn",
  "output": [
    "infer_output"
  ],
  "output_map": {
    "dets": "#0",
    "labels": "#1",
    "masks": "#2"
  },
  "type": "Task"
}
[2023-01-11 23:44:08.943] [mmdeploy] [error] [net_module.cpp:47] Failed to create Net backend: torchscript, config: {
  "context": {
    "device": "<any>",
    "model": "<any>",
    "stream": "<any>"
  },
  "input": [
    "prep_output"
  ],
  "input_map": {
    "img": "#0"
  },
  "is_batched": false,
  "module": "Net",
  "name": "maskrcnn",
  "output": [
    "infer_output"
  ],
  "output_map": {
    "dets": "#0",
    "labels": "#1",
    "masks": "#2"
  },
  "type": "Task"
}
[2023-01-11 23:44:08.945] [mmdeploy] [error] [task.cpp:99] error parsing config: {
  "context": {
    "device": "<any>",
    "model": "<any>",
    "stream": "<any>"
  },
  "input": [
    "prep_output"
  ],
  "input_map": {
    "img": "#0"
  },
  "is_batched": false,
  "module": "Net",
  "name": "maskrcnn",
  "output": [
    "infer_output"
  ],
  "output_map": {
    "dets": "#0",
    "labels": "#1",
    "masks": "#2"
  },
  "type": "Task"
}

**Comment by 1095788063** [comment]
![image](https://user-images.githubusercontent.com/29217820/211851418-4c052cab-139c-4ea2-9879-454bcdb649a6.png)

**Comment by irexyc** [comment]
Have you built torchvision with `-DWITH_CUDA=ON` ?
